pipeline/report.go

```diff
@@ -160,6 +160,15 @@ func (pc *PipelineConfig) reports(reportChan chan *PipelinePack) {
 		reportChan <- pack
 	}
 
+	for _, runner := range pc.allSplitters {
+		pack = <-pc.reportRecycleChan
+		message.NewStringField(pack.Message, "name", runner.Name())
+		message.NewStringField(pack.Message, "key", "splitters")
+		pack.Message.SetLogger(HEKA_DAEMON)
+		pack.Message.SetType("heka.plugin-report")
+		reportChan <- pack
```

Comment:
In here you want to check to see if the splitter supports the `ReportingPlugin` interface, calling `splitter.ReportMsg(pack.Message)` if so.